assets/price-per-item.js

```diff
@@ -71,10 +75,14 @@ if (!customElements.get('price-per-item')) {
         // updatedCartQuantity is not undefined when qty is updated in cart. We need to sum qty in cart and min qty for product.
         this.currentQtyForVolumePricing = updatedCartQuantity === undefined ? this.getCartQuantity(updatedCartQuantity) + enteredQty : this.getCartQuantity(updatedCartQuantity) + parseInt(this.input.min);
 
+        if (this.classList.contains('variant-item__price-per-item')) {
+          this.currentQtyForVolumePricing = this.getCartQuantity(updatedCartQuantity);
+        }
+
         for (let pair of this.qtyPricePairs) {
           if (this.currentQtyForVolumePricing >= pair[0]) {
-            const pricePerItemCurrent = document.querySelector(`price-per-item[id^="Price-Per-Item-${this.dataset.sectionId}"] .price-per-item`);
-            pricePerItemCurrent.innerHTML = pair[1];
+            const pricePerItemCurrent = document.querySelector(`price-per-item[id^="Price-Per-Item-${this.dataset.sectionId || this.dataset.variantId}"] .price-per-item span`);
+            pricePerItemCurrent.innerHTML = window.variantListStrings.each.replace('[money]', pair[1])
```

Comment:
There's an issue with the string displayed on the page after quantity is updated in the Quick order list.

![Screenshot 2023-07-07 at 12 52 58 PM png2](https://github.com/Shopify/dawn/assets/3710548/cfd61e47-4cc8-4ff0-b450-44fb5fa1c43f)